docker/examples/docker-compose-mem.yml

```diff
@@ -20,7 +20,7 @@
 #
 # Note that this file is meant for learning Zipkin, not production deployments.
 
-version: '2'
+version: '2.4'
```

Comment:
this version is what allows health conditions (note: v3 does not!!)

Reply:
just in case, is there any docs confirming this (v3 not supporting healthchecks)? I wasn't able to find this restriction.

Reply:
I actually integrated this with a pinot image for test purposes, and it works in 2.x like so

```yaml
    depends_on:
      kafka-zookeeper:
        condition: service_healthy
```

The docs are very bad about this being removed, only mildly discusses it at the entrypoint of 'condition'
https://docs.docker.com/compose/compose-file/#depends_on
![Screen Shot 2020-08-05 at 7 40 22 PM](https://user-images.githubusercontent.com/64215/89408811-9331e700-d753-11ea-9998-a8fdcf671c50.png)

Options for v3 are not incredible.. basically you have to use wait-for-it and duplicate the health check code of your upstream. I hope v2 lives forever

Reply:
make sense. thanks @adriancole !